Doc/library/dis.rst

```diff
@@ -328,13 +328,16 @@ operation is being performed, so the intermediate analysis object isn't useful:
    source line information (if any) is taken directly from the disassembled code
    object.
 
-   The *show_caches* and *adaptive* parameters work as they do in :func:`dis`.
+   The *adaptive* parameter works as it does in :func:`dis`.
 
    .. versionadded:: 3.4
 
    .. versionchanged:: 3.11
       Added the *show_caches* and *adaptive* parameters.
 
+   .. versionchanged:: 3.13
+      The *show_caches* parameter is deprecated and has no effect.
```

Comment:
Do you plan to add an equivalent feature in future PR?

Reply:
Let me clarify the comment here. *show_caches* still works in all the API functions of dis, except here in ``get_instructions``, which returns the instruction stream but doesn't really show anything. It still gives the cache info, but attached to the instructions rather than as separate fake instructions, and it does this regardless of the value of *show_caches*.